Source/View Controller/LiveUpdatesTableViewController.swift

```diff
@@ -26,15 +26,15 @@ internal class LiveUpdatesTableViewController: NavigationBarTableViewController,
         
         self.initLiveCountDown()
         self.navigationItem.largeTitleDisplayMode = .never
-        self.colorUpper(view: tableView, with: BACKGROUND_COLOR)
+        self.colorUpper(view: tableView, with: LIVE_UPDATES_MENU_COLOR)
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
         self.attachRefreshControl()
         self.viewModel.fetchRecent()
-        self.add(navigationController: navigationController, and: navigationItem, with: BACKGROUND_COLOR)
+        self.add(navigationController: navigationController, and: navigationItem, with: LIVE_UPDATES_MENU_COLOR)
```

Comment:
These are suppose to remain the background color not change to match the menu color